### PR TITLE
Dev/rzepi/flexbox

### PR DESCRIPTION
**Pull request by RZEPI**

### CODE REVIEW

**Review by grczk — DISMISSED** [review]

File: vue-app/src/models/LayoutProps.ts
Comment (marziel2001) [inline vue-app/src/models/LayoutProps.ts:1]: Why export just a string? ;)

**Review by marziel2001 — DISMISSED** [review]

**Review by marziel2001 — DISMISSED** [review]

File: vue-app/src/pages/FlexboxPage.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<template>
+  <base-layout title="Flexbox Generator">
```
Comment (grczk) [inline vue-app/src/pages/FlexboxPage.vue]: on the react page it is called "Flexbox Testing"

File: vue-app/src/components/UI/SidebarLink.vue [hunk 2]
```diff
@@ -26,17 +33,29 @@ a {
   display: grid;
   grid-template-columns: 1fr 2fr 4fr 1fr;
 }
+
 li {
   list-style-type: none;
   width: 100%;
+  max-height: 5vh;
```
Comment (grczk) [inline vue-app/src/components/UI/SidebarLink.vue:40]: This property causes the list items to overlap.
The same issue exists on the react page, also due to the max-height.
image

**Review by grczk — CHANGES_REQUESTED** [review]

File: vue-app/src/pages/FlexboxPage.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    import("../components/flexbox/FlexboxPreview.vue"),
+    loadingComponent: LoadingFallback,
+    delay: 0,
+}
+);
+</script>
+
+<style scoped>
+.main-container {
+  --flex-element-size: 6em;
```
Comment (grczk) [inline vue-app/src/pages/FlexboxPage.vue]: on the react page the value is 0.4em

**Review by grczk — CHANGES_REQUESTED** [review]

File: vue-app/src/components/UI/SidebarLink.vue (hunk 2 above)
Comment (RZEPI) [reply vue-app/src/components/UI/SidebarLink.vue:40]: this is not issue for this branch, I will create another for the fix for that

**Review by marziel2001 — APPROVED** [review]

**Review by grczk — APPROVED** [review]

**Review by marziel2001 — CHANGES_REQUESTED** [review]
I cannot open the page. My browser freezes completely, and F12 doesn't work

**Review by marziel2001 — APPROVED** [review]

**Pull request merged**